homeassistant/components/nobo_hub/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Nob\u00f8 Ecohub",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/nobo_hub",
-  "requirements": ["pynobo==1.3.0"],
+  "requirements": ["pynobo==1.4.0"],
```

Comment:
Side note: The event loop is not directly needed to create a task. Just use `asyncio.create_task`.

https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task

https://github.com/echoromeo/pynobo/blob/026e0c41776822a53b2c43546ebd4c2dd0d3b603/pynobo.py#L394-L395

Reply:
Thanks - I will make a PR for `pynobo` with this simplification.

The implementation of `asyncio.create_task` is equivalent though, so it shouldn't be a blocker for this PR?

https://github.com/python/cpython/blob/main/Lib/asyncio/tasks.py#L371-L384

Reply:
Ni, I just noticed it when looking at the library and wanted to mention it.

Reply:
Is fixed and will be in the next release of `pynobo`.